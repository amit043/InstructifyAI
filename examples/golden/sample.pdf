%PDF-1.4
%
4 0 obj
<</Type /Page
/Parent 3 0 R
/MediaBox [0 0 595 842]
/Contents 5 0 R/Group <</CS 6 0 R>>>>>
endobj
5 0 obj
<</Length 55>>stream
BT
/F1 30 Tf
100 708 Tj
( hi dath) Tj

F1 10 Tf
100 688 Tf

ET
endstream
endobj
1 0 obj
<</Type /Catalog
/Pages 2 0 R >>
endobj
2 0 obj
<</Type /Pages
/Kids [3 0 R]
/Count 1 >>
endobj
3 0 obj
<</Type /Page
/Parent 2 0 R
/MediaBox [0 0 595 842]
/Resources <</Font <</F1 4 0 R>>>>
/Contents 5 0 R/Group <</CS 6 0 R>>>>>
endobj
4 0 obj
<</Type /Font
/Subtype /Type1
/BaseFont /Times-Roman>>
endobj
6 0 obj
<</Type /GROUP
/S/Transparency
>>
endobj
xref
0 7
0000000000 65535 f 
0000000083 00000 n 
0000000159 00000 n 
0000000205 00000 n 
0000000304 00000 n 
0000000353 00000 n 
0000000423 00000 n 
trailer
<</Size 7/Root 1 0 R/Info 7 0 R>>
startxref
419
%%EOF