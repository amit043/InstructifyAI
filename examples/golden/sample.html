<html>
  <body>
    <nav>ignore</nav>
    <h1>Title</h1>
    <p>First paragraph text.</p>
    <h2>Section A</h2>
    <p>Second paragraph with more words.</p>
    <table><tr><td>1</td></tr></table>
    <p>After table paragraph.</p>
    <pre><code>print('code block')</code></pre>
    <footer>ignore</footer>
  </body>
</html>
